src/Utils/GeneralComponents/Countdown.js

```diff
@@ -0,0 +1,32 @@
+import { useState, useEffect } from "react";
+import moment from "moment";
+import "moment-duration-format";
+export default function Countdown(startTime, endTime) {
+  const [duration, setDuration] = useState(
+    moment.duration(endTime - startTime)
+  );
+  const [zero, setZero] = useState(false);
```

Comment:
we will talk about it more at the higher levels, but for now - I believe that `hasReachedZero` and `setHasReachedZero` are better names